reactos/drivers/usb/usbhub_new/debug.c

```diff
@@ -45,7 +45,7 @@ USBPORT_DumpingConfiguration(IN PUSB_CONFIGURATION_DESCRIPTOR ConfigDescriptor)
 
     Descriptor = (PUSB_COMMON_DESCRIPTOR)ConfigDescriptor;
 
-    do
+    while (Descriptor->bLength)
```

Comment:
That's not exactly the while condition I had in mind. In general the pattern

```
do
{
    if (condition) break;
    ...
} while (1);
```

is identical to

```
while (condition)
{
   ...
}
```

And since the latter is easier to read, I tend to ask you to change it. This usually does not mean I'm asking for a change in logic.

That said, the logic change here might make sense, I'll double-check on the next pass.